contracts/VoteWeighting.sol

```diff
@@ -683,30 +685,44 @@ contract VoteWeighting {
         return setNominees.length - 1;
     }
 
+    /// @dev Get the total number of removed nominees.
+    /// @notice The zero-th default nominee Id with id == 0 does not count.
+    /// @return Total number of removed nominees.
+    function getNumRemovedNominees() external view returns (uint256) {
+        return setRemovedNominees.length - 1;
+    }
+
     /// @dev Gets a full set of nominees.
     /// @notice The returned set includes the zero-th empty nominee instance.
-    /// @return nominees Set of all the nominees in the contract.
-    function getAllNominees() external view returns (Nominee[] memory nominees) {
-        nominees = setNominees;
+    /// @return Set of all the nominees in the contract.
+    function getAllNominees() external view returns (Nominee[] memory) {
+        return setNominees;
+    }
+
+    /// @dev Gets a full set of removed nominees.
+    /// @notice The returned set includes the zero-th empty nominee instance.
+    /// @return Set of all the removed nominees in the contract.
+    function getAllRemovedNominees() external view returns (Nominee[] memory) {
```

Comment:
Simple getter for the set of removed nominees.

Reply:
awesome, thank you